docs/reference/dependencies.md

```diff
@@ -622,309 +622,272 @@ import TabItem from "@theme/TabItem";
+- [com.fasterxml.jackson.module:jackson-module-parameter-names:2.13.4](https://github.com/FasterXML/jackson-modules-java8/jackson-module-parameter-names) (The Apache Software License, Version 2.0)
+- [com.github.fge:btf:1.2](https://github.com/fge/btf) (Apache Software License, version 2.0) (Lesser General Public License, version 3 or greater)
+- [com.github.fge:jackson-coreutils:1.6](https://github.com/fge/jackson-coreutils) (Apache Software License, version 2.0) (Lesser General Public License, version 3 or greater)
+- [com.github.fge:json-patch:1.9](https://github.com/fge/json-patch) (Apache Software License, version 2.0) (Lesser General Public License, version 3 or greater)
+- [com.github.fge:msg-simple:1.1](https://github.com/fge/msg-simple) (Apache Software License, version 2.0) (Lesser General Public License, version 3 or greater)
+- [com.github.stephenc.jcip:jcip-annotations:1.0-1](http://stephenc.github.com/jcip-annotations) (Apache License, Version 2.0)
+- [com.google.code.findbugs:jsr305:3.0.2](http://findbugs.sourceforge.net/) (The Apache Software License, Version 2.0)
+- [com.google.errorprone:error_prone_annotations:2.3.4](http://nexus.sonatype.org/oss-repository-hosting.html/error_prone_parent/error_prone_annotations) (Apache 2.0)
+- [com.google.guava:failureaccess:1.0.1](https://github.com/google/guava/failureaccess) (The Apache Software License, Version 2.0)
+- [com.google.guava:guava:30.0-jre](https://github.com/google/guava/guava) (Apache License, Version 2.0)
+- [com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava](https://github.com/google/guava/listenablefuture) (The Apache Software License, Version 2.0)
+- [com.google.j2objc:j2objc-annotations:1.3](https://github.com/google/j2objc/) (The Apache Software License, Version 2.0)
+- [com.squareup.okhttp3:logging-interceptor:4.9.3](https://square.github.io/okhttp/) (The Apache Software License, Version 2.0)
+- [com.squareup.okhttp3:okhttp:4.9.3](https://square.github.io/okhttp/) (The Apache Software License, Version 2.0)
+- [com.squareup.okio:okio:2.8.0](https://github.com/square/okio/) (The Apache Software License, Version 2.0)
+- [com.sun.activation:jakarta.activation:1.2.2](https://github.com/eclipse-ee4j/jaf/jakarta.activation) (EDL 1.0)
+- [com.sun.istack:istack-commons-runtime:3.0.12](https://projects.eclipse.org/projects/ee4j/istack-commons/istack-commons-runtime) (Eclipse Distribution License - v 1.0)
+- [com.sun.mail:jakarta.mail:1.6.7](http://eclipse-ee4j.github.io/mail/jakarta.mail) (EDL 1.0) (EPL 2.0) (GPL2 w/ CPE)
+- [commons-codec:commons-codec:1.15](https://commons.apache.org/proper/commons-codec/) (Apache License, Version 2.0)
+- [commons-io:commons-io:2.5](http://commons.apache.org/proper/commons-io/) (Apache License, Version 2.0)
+- [dev.failsafe:failsafe:3.3.0](https://failsafe.dev/failsafe) (Apache License, Version 2.0)
+- io.camunda:identity-sdk:8.0.0-SNAPSHOT (Unknown license)
```

Comment:
I'm not sure how we should handle this dependency on ourselves

Reply:
All sub dependencies of the SDK should also be included in this list, right? In this case we can just remove this entry.

Reply:
It appears that way yes, so I've removed this line (and the corresponding line in the versioned docs too)